android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/RibRefWatcher.kt

```diff
@@ -39,12 +39,15 @@ public open class RibRefWatcher {
    *
    * @param objectToWatch the object to watch.
    */
-  public open fun watchDeletedObject(objectToWatch: Any?) {
+  public open fun watchDeletedObject(
+    objectToWatch: Any?,
+    description: String
```

Comment:
Should be an optional param to avoid being a breaking API change for consumers

Reply:
Done by adding a deprecated overload without the param (that way it signifies clearly that the call should be updated)